chroma-manager/tests/integration/shared_storage_configuration/test_power_control.py

```diff
@@ -178,3 +178,9 @@ def boot_time_is_newer():
             return post_boot_time > pre_boot_time
 
         self.wait_until_true(boot_time_is_newer)
+
+        # remove any corrupted dnf lock files https://github.com/intel-hpdd/intel-manager-for-lustre/issues/618
+        self.remote_command(
+            self.server['address'],
+            '[[ -f {0} && ! -s {0} ]] && rm -f {0}'.format('/var/cache/dnf/metadata_lock.pid'),
```

Comment:
If we can identify that this lock file really is the result of taking the rug out from under a node (which is what this patch is implying), we'd be better to do this kind of cleanup on node startup since it's potentially production affecting.